app/Filament/Shared/RelationManagers/MembersRelationManager.php

```
@@ -0,0 +1,87 @@
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Schemas\Schema;
+use Filament\Tables\Columns\IconColumn;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Model;
+
+class MembersRelationManager extends RelationManager
+{
+    protected static string $relationship = 'members';
+
+    public static function getBadge(Model $ownerRecord, string $pageClass): ?string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$pageClass'. (undefined)

(UnusedFormalParameter)